wfl/descriptors/calc.py

```diff
@@ -8,7 +8,7 @@
 except ModuleNotFoundError:
```

Comment:
Why not to rename this to `wfl/descriptors.py` or something similar? Especially because there is a `calc` function there already??

Reply:
Now I'm confused - this used to be `wfl/descriptors.py`  @gabor1 do you remember why we said we should move it into a subdirectory, given that there isn't anything else in `wfl/descriptors/`?

Reply:
renamed to `descriptors/quippy.py`